tests/integrations/mcs/tso/server_test.go

```diff
@@ -158,6 +158,10 @@ func (suite *tsoServerTestSuite) TestParticipantStartWithAdvertiseListenAddr() {
 
 func TestTSOPath(t *testing.T) {
 	re := require.New(t)
+	re.NoError(failpoint.Enable("github.com/tikv/pd/server/skipKeyspaceRegionCheck", "return"))
```

Comment:
Why does it need to add this fail point?

Reply:
If we don't add it, it will meet `Load keyspace DEFAULT failed: type:ENTRY_NOT_FOUND message:"[PD:keyspace:ErrKeyspaceNotFound]keyspace does not exist" `